gdk4-wayland/sys/src/lib.rs

```diff
@@ -134,10 +134,6 @@ impl ::std::fmt::Debug for GdkWaylandToplevel {
 
 #[link(name = "gtk-4")]
 extern "C" {
-
-    //=========================================================================
-    // GdkWaylandDevice
-    //=========================================================================
```

Comment:
regen checker would complain about this

Reply:
Just updated my rustfmt and force-pushed. ;)